baybe/surrogates/bandit.py

```diff
@@ -0,0 +1,153 @@
+"""Multi-armed bandit surrogates."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, ClassVar
+
+from attrs import define, field
+
+from baybe.exceptions import IncompatibleSearchSpaceError, ModelNotTrainedError
+from baybe.parameters.categorical import CategoricalParameter
+from baybe.parameters.enum import CategoricalEncoding
+from baybe.priors import BetaPrior
+from baybe.searchspace.core import SearchSpace
+from baybe.surrogates.base import Surrogate
+from baybe.targets.binary import _NEGATIVE_VALUE_COMP, _POSITIVE_VALUE_COMP
+from baybe.utils.random import temporary_seed
+
+if TYPE_CHECKING:
+    from botorch.models.model import Model
+    from botorch.posteriors import TorchPosterior
+    from torch import Tensor
+
+
+@define
+class BernoulliMultiArmedBanditSurrogate(Surrogate):
+    """A multi-armed bandit model with Bernoulli likelihood and beta prior."""
```

Comment:
Hey @AVHopp, would like to help/adjust here, but I really don't know how. Can you make your request more specific? For the two particular points you mentioned, I don't know how I can improve tbh, because the info would rather belong the base class:
* The number of arms for the bandit is equivalent to the number of dimensions for the GP. Also there, we don't give any specifics. The surrogates simply adjust to whatever they are trained on. I agree that we can talk more about this in general in the base class, bit this is out of scope for the PR
* What about the Prior? It's a hard-coded beta prior. So don't know what to add here.

Reply:
Follow-up: class will be restructured in immediate follow-up PR (provided that both of you still want that). Therefore, the final class docstring should be defined there. Resolving here